src/xr/xr-hand.js

```diff
@@ -147,27 +147,51 @@ XrHand.prototype.update = function (frame) {
         this._inputSource._rayLocal.direction.lerp(vecC, vecA, 0.5).normalize();
     }
 
-    // emulate select events by touching thumb tip and index tips
-    if (j4 && j9) {
-        vecA.copy(j4._localPosition);
-        var d = vecA.distance(j9._localPosition);
-
-        if (d < 0.015) { // 15 mm
-            if (! this._inputSource._selecting) {
-                this._inputSource._selecting = true;
-                this._inputSource.fire('selectstart');
-                this._manager.input.fire('selectstart', this._inputSource);
+    // emulate squeeze events by folding all 4 fingers
+    var squeezing = false;
+    var squeezeTolerance = -0.8;
+
+    // first finger
+    vecA.sub2(this._fingers[1].joints[0]._localPosition, this._fingers[1].joints[1]._localPosition).normalize();
```

Comment:
maybe you should wrap this code in a function and call it 4 times, once for each finger?